libr/core/project.c

```diff
@@ -159,6 +159,56 @@ R_API int r_core_project_delete(RCore *core, const char *prjfile) {
 	return 0;
 }
 
+R_API bool r_core_rop_load(RCore *core, const char *prjfile) {
+	char *path, *db;
+	ut8 found = 0;
+	SdbListIter *it;
+	SdbNs *ns;
+
+	Sdb *rop_db = sdb_ns (core->sdb, "rop", false);
+
+	if (!prjfile || !*prjfile) {
+		return false;
+	}
+
+	if (prjfile[0] == '/') {
+		db = r_str_newf ("%s.d", prjfile);
+		if (!db) return false;
+		path = strdup (db);
+	} else {
+		db = r_str_newf (R2_HOMEDIR"/projects/%s.d", prjfile);
```

Comment:
Theres a variable already for this 

> On 29 Jul 2016, at 21:27, Jeffrey Crowell <notifications@github.com> wrote:
> 
> In libr/core/project.c:
> 
> > +	ut8 found = 0;
> > +	SdbListIter *it;
> > +	SdbNs *ns;
> > +
> > +	Sdb *rop_db = sdb_ns (core->sdb, "rop", false);
> > +
> > +	if (!prjfile || !*prjfile) {
> > +		return false;
> > +	}
> > +
> > +	if (prjfile[0] == '/') {
> > +		db = r_str_newf ("%s.d", prjfile);
> > +		if (!db) return false;
> > +		path = strdup (db);
> > +	} else {
> > +		db = r_str_newf (R2_HOMEDIR"/projects/%s.d", prjfile);
> really, we should have a FS-independent r_path_append or something.
> that checks if we're on windows to use the backslashes.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>